WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -335,6 +338,20 @@ final class ShippingLabelFormViewModel {
 
         return price
     }
+
+    /// Filter country for picking based on ship type.
+    ///
+    /// For origin address, country list should show only US or any of its territories that have at least one USPS postal office.
+    /// Destination address should allow picking from the complete country list.
+    ///
+    func filteredCountries(for type: ShipType) -> [Country] {
+        switch type {
+        case .origin:
+            return countries.filter { Constants.acceptedUSPSCountries.contains($0.code) }
```

Comment:
Probably good enough for now, but what will be the strategy to keep these in sync as `UPSP` or `Core` updates this list of countries?

I wonder if we should be fetching this remotely? 🤔

Reply:
This is a valid point! I will bring this question to Slack and see what the team suggests 👍